README.md

```diff
@@ -1,60 +1,92 @@
-# Source Code Annotation application
+[![Build Status](https://travis-ci.org/src-d/code-annotation.svg)](https://travis-ci.org/src-d/code-annotation)
+![unstable](https://svg-badge.appspot.com/badge/stability/unstable?a)
+
+# Source Code Annotation Tool
 
 In order to evaluate quality of ML models, as well as to create “ImageNet for source core” there is a need for tools to automate the data collection/labeling/annotation.
 
+![Screenshot](.github/screenshot.png?raw=true)
+
 ## Installation
 
 ### Github OAuth tokens
 
-First you need OAuth application on github. [Read how to create it](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
+1. You need OAuth application on github. [Read how to create it](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
```

Comment:
Try to write links that describe what they point to.

You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](...).